component/src/main/java/com/fioalpha/poc/component/view/ResultInvestmentMedium.kt

```diff
@@ -1,33 +1,28 @@
 package com.fioalpha.poc.component.view
 
 import android.content.Context
-import android.text.Editable
-import android.text.TextWatcher
 import android.util.AttributeSet
 import android.view.LayoutInflater
-import android.widget.EditText
 import android.widget.LinearLayout
 import com.fioalpha.poc.component.databinding.InvestmentResultMediumBinding
 import com.fioalpha.poc.component.extension.setMonetary
 import com.fioalpha.poc.component.extension.setMoneyWithTaxes
 import com.fioalpha.poc.component.view.model.ResultInvestmentMediumModel
-import java.text.NumberFormat
 import java.util.*
 
-
 class ResultInvestmentMedium @JvmOverloads constructor(
     context: Context,
     attributeSet: AttributeSet? = null,
     defStyleAttr: Int = 0
-): LinearLayout(context, attributeSet, defStyleAttr) {
+) : LinearLayout(context, attributeSet, defStyleAttr) {
 
-    private val viewBinding:InvestmentResultMediumBinding by lazy {
+    private val viewBinding: InvestmentResultMediumBinding by lazy {
         InvestmentResultMediumBinding.inflate(LayoutInflater.from(context))
     }
 
     init {
        addView(
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/fioalpha/CaseStudy/blob/6ce9cd99c0c1ee323fc1eba2aae62b43356b0111/component/src/main/java/com/fioalpha/poc/component/view/ResultInvestmentMedium.kt#L24"></span>Unexpected indentation (7) (it should be 8) (cannot be auto-corrected)</td>
    </tr>
  </tbody>
</table>